cmd/ipfs-cluster-ctl/main.go

```diff
@@ -501,6 +510,10 @@ content.
 						formatResponse(c, lastBuf, nil)
 						return
 					}
+					if c.Bool("wait") {
```

Comment:
There are `return` statements above that need to be removed so that the code-flow always hits the `wait` part. Right now it will disregard the wait flag if --quieter or --no-stream are passed, for example.

Reply:
uff. I should not push code in the evening. fixed!